src/crimson/os/seastore/cache.cc

```diff
@@ -1475,6 +1520,35 @@ void Cache::complete_commit(
       ceph_abort("not possible");
     }
   }
+
+  auto existing_stats = t.get_existing_block_stats();
+  DEBUGT("total existing blocks num: {}, exist clean num: {}, "
+	 "exist mutation pending num: {}",
+	 t,
+	 existing_stats.valid_num,
+	 existing_stats.clean_num,
+	 existing_stats.mutated_num);
+  for (auto &i: t.existing_block_list) {
+    if (i->is_valid()) {
+      if (i->is_exist_clean()) {
+	i->state = CachedExtent::extent_state_t::CLEAN;
+      } else if (i->is_exist_mutation_pending()) {
+	i->state = CachedExtent::extent_state_t::DIRTY;
+      }
```

Comment:
```
      } else {
        assert(0 == "must be EXIST_CLEAN or EXIST_MUTATION_PENDING");
      }
```
?

Reply:
yes, state must be EXIST_CLEAN or EXIST_MUTATION_PENDING.
done.

Reply:
at this moment EXIST_MUTATION_PENDING extents have become DIRTY(in mutated_block_list).
so its state must be EXIST_CLEAN or DIRTY.

Reply:
Oops, sorry

```
      } else {
        assert(0 == "must be EXIST_CLEAN or DIRTY");
      }
```

Still, seems like it would benefit from an assert?

Reply:
```
      if (i->is_exist_clean()) {
        i->state = CachedExtent::extent_state_t::CLEAN;
      } else {
        assert(i->state == CachedExtent::extent_state_t::DIRTY);
      }
```
done.